owncloudData/src/androidTest/java/com/owncloud/android/data/roommigrations/MigrationToDB36Test.kt

```diff
@@ -0,0 +1,113 @@
+                    "version_string, " +
+                    "version_edition, " +
+                    "core_pollinterval, " +
+                    "dav_chunking_version, " +
+                    "sharing_api_enabled, " +
+                    "sharing_public_enabled, " +
+                    "sharing_public_password_enforced, " +
+                    "sharing_public_password_enforced_read_only, " +
+                    "sharing_public_password_enforced_read_write, " +
+                    "sharing_public_password_enforced_public_only, " +
+                    "sharing_public_expire_date_enabled, " +
+                    "sharing_public_expire_date_days, " +
+                    "sharing_public_expire_date_enforced, " +
+                    "sharing_public_upload, " +
+                    "sharing_public_multiple, " +
+                    "supports_upload_only, " +
+                    "sharing_resharing, " +
+                    "sharing_federation_outgoing, " +
+                    "sharing_federation_incoming, " +
+                    "files_bigfilechunking, " +
+                    "files_undelete, " +
+                    "files_versioning)" +
+                    " VALUES " +
+                    "(?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)",
+            arrayOf(
+                OC_CAPABILITY.accountName,
+                OC_CAPABILITY.versionMayor,
+                OC_CAPABILITY.versionMinor,
+                OC_CAPABILITY.versionMicro,
+                OC_CAPABILITY.versionString,
+                OC_CAPABILITY.versionEdition,
+                OC_CAPABILITY.corePollInterval,
+                OC_CAPABILITY.davChunkingVersion,
+                OC_CAPABILITY.filesSharingApiEnabled,
+                OC_CAPABILITY.filesSharingPublicEnabled,
+                OC_CAPABILITY.filesSharingPublicPasswordEnforced,
+                OC_CAPABILITY.filesSharingPublicPasswordEnforcedReadOnly,
+                OC_CAPABILITY.filesSharingPublicPasswordEnforcedReadWrite,
+                OC_CAPABILITY.filesSharingPublicPasswordEnforcedUploadOnly,
+                OC_CAPABILITY.filesSharingPublicExpireDateEnabled,
+                OC_CAPABILITY.filesSharingPublicExpireDateDays,
+                OC_CAPABILITY.filesSharingPublicExpireDateEnforced,
+                OC_CAPABILITY.filesSharingPublicUpload,
+                OC_CAPABILITY.filesSharingPublicMultiple,
+                OC_CAPABILITY.filesSharingPublicSupportsUploadOnly,
+                OC_CAPABILITY.filesSharingResharing,
+                OC_CAPABILITY.filesSharingFederationOutgoing,
+                OC_CAPABILITY.filesSharingFederationIncoming,
+                OC_CAPABILITY.filesBigFileChunking,
+                OC_CAPABILITY.filesUndelete,
+                OC_CAPABILITY.filesVersioning
+            )
+        )
+    }
+
+    private fun validateMigrationTo36(database: SupportSQLiteDatabase) {
+        val backUp = getCount(database, CAPABILITIES_TABLE_NAME)
```

Comment:
Let's call it `val capabilityCount = ....` 👍

Reply:
I agree with you, it's more understandable.